components/library/FreeImage/pkg5

```diff
@@ -6,7 +6,7 @@
         "image/library/libraw",
         "image/library/libtiff",
         "image/library/openjpeg",
-        "library/ilmbase",
+        "library/imath",
         "library/jxrlib",
```

Comment:
How is the dependency on imath detected on the dependent packages? Do they need some "help", eg. by new flags?